scraper/src/youtube2zim/youtube.py

```diff
@@ -345,7 +345,7 @@ def extract_playlists_details_from(collection_type, youtube_id):
         raise NotImplementedError("unsupported collection_type")
 
     return (
-        [Playlist.from_id(playlist_id) for playlist_id in list(set(playlist_ids))],
+        [Playlist.from_id(playlist_id) for playlist_id in dict.fromkeys(playlist_ids)],
```

Comment:
I'd add a comment here mentioning intent as fact that dict is now order-keeping since is not widespread

Reply:
Added in [2a93139](https://github.com/openzim/youtube/pull/296/commits/2a9313962f7745ced3e42c0fbe8d3f65440a1ab7)